src/PrestaShopBundle/Controller/Admin/Improve/ModuleController.php

```diff
@@ -476,6 +472,10 @@ public function moduleAction(Request $request)
                 ]
             );
 
+            // Get accessed module object
+            $moduleAccessed = $moduleRepository->getModule($module);
```

Comment:
Could you explain why this change is needed?

Reply:
@PierreRambaud because in the install action, [`getModule` is called]((https://github.com/PrestaShop/PrestaShop/blob/develop/src/Core/Addon/Module/ModuleManager.php#L310)) after the module has been downloaded, but if you call this method before, the data is cached and the next time you call it, it won't take the fact that the module is now on the disk into account, leading to an error.